Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -1039,6 +1230,253 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
       return new Rect(r.x + padding, r.y + padding, r.width - (padding * 2), r.height - (padding * 2));
     }
 
+    /// <summary>
+    /// Returns a new Rect with the argument padding as an outward margin on each border
+    /// of this Rect; the result is a larger Rect.
+    /// </summary>
+    public static Rect PadOuter(this Rect r, float width) {
```

Comment:
Would change the name of the argument to `padding` instead of `width` to make sure there is no misunderstanding that it only affects the rect in the X direction.

Reply:
Also I see now that this does something conceptually different compared to PadTop/PadBottom....   maybe could be named something different to be super explicit?  Maybe Extrude instead?

Reply:
yeah can do